### ISSUE TITLE
R version in DESCRIPTION

### ISSUE THREAD

**Issue by pimi17** [Aufiero/circRNAprofiler#7]
Hey guys,

I do not want to bother you, but I believe that is a typo in the DESCRIPTION file.
On the Depends section, there is mentioned R version 4.1.0.
```
Depends: 
    R(>= 4.1.0)
```
At the current time, the latest release is 4.0.4 and 4.1.0 is still under development.
I need to clarify this because I want to add the latest release of circRNAprofiler to the Guix package manager.

Thanks for your time!

**Comment by Aufiero** [comment]
Dear Mădălin,

I checked and everything should be correct. 

- the Release version of circRNAprofiler (3.12) depends on R>= 4.0.0 (latest release at the moment)
( see https://bioconductor.org/packages/release/bioc/html/circRNAprofiler.html)
To install use: 
```
if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")

BiocManager::install("circRNAprofiler")
```

- the Development version of circRNAprofiler (3.13) depends on R>= 4.1.0  that is the R-devel (both will be released mid-Arpril) 
( see https://bioconductor.org/packages/devel/bioc/html/circRNAprofiler.html)
To install use: 
```
if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")

# The following initializes usage of Bioc devel
BiocManager::install(version='devel')

BiocManager::install("circRNAprofiler")
```
NOTE: On Github there is the Development version of circRNAprofiler (3.13), you should always use (when possible) BiocManager to install R packages.

FYI  read on https://bioconductor.org/developers/how-to/useDevel/:
Package authors should develop against the version of R that will be available to users when the Bioconductor devel branch becomes the Bioconductor release branch.R has a ‘.y’ release in x.y.z every year (typically mid-April), but Bioconductor has a .y release (where current devel becomes release) every 6 months (mid-April and mid-October).This means that, from mid-October through mid-April, Bioconductor developers should be developing against R-devel.
Does this resolve your concerns?  

Best, S

**Comment by pimi17** [comment]
Dear S,

Many thanks for the explanations!
We added version 1.4.0 to Guix!
https://guix.gnu.org/en/packages/r-circrnaprofiler-1.4.0/

Sorry for my delayed response and many thanks, 
                                                              Mădălin!

**Comment by Aufiero** [comment]
Dear Mădălin,

I had to fix a bug on release 1.4.0, so please use the release version 1.4.2, which passed the build and check on Bioconductor. 

Thanks.
Simona
